lib/widgets/loading_animated_text.dart

```diff
@@ -39,7 +39,7 @@ class LoadingAnimatedTextState extends State<LoadingAnimatedText> {
   Widget build(BuildContext context) {   
     return RichText(
       text: TextSpan(
-        style: DefaultTextStyle.of(context).style,
+        style: widget.textStyle == TextStyle() ? DefaultTextStyle.of(context).style : widget.textStyle,
```

Comment:
What is the purpose of this? Can you explain?

Reply:
Previously, LoadingAnimatedText never used the value of textStyle parameter. We're using DefaultTextStyle.of(context).style if the parameters value is 'TextStyle()' which is default value of the parameter. We could remove the initial value and use 'widget.textStyle ?? DefaultTextStyle.of(context).style'

Reply:
@erdemyerebasmaz in that case I think your suggestion is better and clearer.
Let's remove the initial value.
One caveat though is that I have noticed some usage of LoadingAnimatedText with explicit textStyle passed from outside and as we see now in the code this style was never applied... Now with the change we suggesting the style will be used (as it should) but we need to make sure it looks good and creates any regression.